docker.go

```diff
@@ -89,6 +90,28 @@ func getIPsToNames(client dockerClienter, id string) (ipsToNamesMap, error) {
 			names = appendNames(names, name)
 		}
 
+		if label, ok := containerFull.Config.Labels["com.costela.docker-etchosts.add_hosts"]; ok {
+			if (strings.HasPrefix(label, "[")) {
```

Comment:
maybe we can be a bit more defensive here?
```suggestion
			label = strings.TrimSpaces(label)
			if (strings.HasPrefix(label, "[")) {
```